frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -178,6 +180,88 @@ export const surveyLogic = kea<surveyLogicType>([
                 editingSurvey: (_, { editing }) => editing,
             },
         ],
+        survey: [
+            { ...NEW_SURVEY } as NewSurvey | Survey,
+            {
+                setDefaultForQuestionType: (state, { type }) => {
+                    if (type === SurveyQuestionType.Open) {
```

Comment:
I think it could be a good idea to refactor this a little since we're likely to re-use this code when doing templates for surveys!

Instead of individual if statements, maybe we could store these "default values" in a default object instead, like 
```
const defaultSurveyTemplates = { 
[SurveyQuestionType.Link]: { questions: ....., appearance: ..... },
 [SurveyQuestionType.Rating]: { ...... },
}```

and then we can just directly call for the key value pairing here instead of each if statement

Reply:
Parking for now, as discussed